locale/fi/pagination.json

```diff
@@ -0,0 +1,56 @@
+{
+  "title": "Sivutus (Pagination)",
+  "intro": "Sivutus on komponentti jonka avulla käyttäjä voi navigoida useille sivuille jaettua sisältöä. Esimerkiksi hakutuloksia.",
+  "note.title": "Saavutettavuus ja käytettävyys",
+  "note.items": [
+    "Ruudunlukijaa varten on erikseen oma teksti sivunumeron tietokenttää varten. Esimerkiksi näkyvän tekstin \"Sivu 1 / 20\" sijaan ruudunlukijalle on parempi antaa teksti \"Sivu 1 kautta 20\"",
```

Comment:
Tämä on mielestäni vähän huono esimerkki, koska ruudunlukija jo lukee tekstin "Sivu 1 / 20" "Sivu 1 kautta 20", jos puhekielenä on suomi.

Reply:
Minulla oli tuon kanssa aiemmin jotain ongelmia, mutta nyt kun kokeilin niin tosiaan myös voiceover lausuu / merkin ihan nätisti. Parempi esimerkki kyllä kelpaisi.

Reply:
Englanniksi tuohon on helpompi keksiä esimerkki kuten "Page 3 / 5"  vs "Page 3 out of 5", mutta suomeksi hankalampi.

Reply:
Voisko tän muotoilla vaikka vain et "Ruudunlukijaa varten informaatio kannattaa antaa puhtaasti tekstimuodossa, esim. "yksi kautta kaksikymmentä", sillä erikoismerkkien lukutapa saattaa vaihdella." tjsp.

Reply:
Numeroita ei varmaankaan kannata selkeyden vuoksi kirjoittaa esimerkkiin auki. Mutta muutoin tuo Joonaksen ehdotus voisi olla hyvä, korostamaan erikoismerkkien lukemista ruudunlukijalla.

Reply:
Siirtäisin myös tän tonne listan alimmaiseksi, niin ois ehkä noi selkeämmät ohjeet eka ja sit tällainen spesifimpi vasta sen jälkeen.

Reply:
Päivitin tekstin ja se on nyt viimeisenä listalla.